models/statistic.go

```diff
@@ -37,7 +44,17 @@ func GetStatistic() (stats Statistic) {
 		Count    int64
 		IsClosed bool
 	}
+
 	issueCounts := []IssueCount{}
+	stats.Counter.IssueByLabel = []IssueByLabelCount{}
+
+	_ = db.GetEngine(db.DefaultContext).
+		Select("COUNT(*) AS count, l.name as label").
+		Join("LEFT", "issue_label il", "i.id=il.issue_id").
```

Comment:
Am I misunderstanding something here about the data structure of the `issue_label` table, or shouldn't that already store the label name with it? Is the second join really needed?

Reply:
`issue_label` join tables `issue` and `label`. You right I added an extra join, I don't need table `issue` so I removed it.